### PR TITLE
Clarify requirement for re-indexing from ES 1.x

### PR DESCRIPTION
**Pull request by peterskim12**

### CODE REVIEW

File: docs/reference/migration/migrate_5_0.asciidoc [hunk 1]
```diff
@@ -24,7 +24,7 @@ version of Elasticsearch before 2.0.
 =========================================
 
 Indices created in Elasticsearch 1.x or before will need to be reindexed with
-Elasticsearch 2.x in order to be readable by Elasticsearch 5.x. It is not
+Elasticsearch 2.x or 5.x in order to be readable by Elasticsearch 5.x. It is not
```
Comment (nik9000) [inline docs/reference/migration/migrate_5_0.asciidoc]: I think it'd be more clear to say s/5.x/5.x's <<reindex-from-remote>>/.

Comment (nik9000) [inline docs/reference/migration/migrate_5_0.asciidoc]: Possessive. I was just thinking it is nice to mention that in order to reindex from 1.x to 5.x you need reindex-from-remote. You can't use regular reindex.

It looks like github ate my attempt to write asciidoc syntax - I've edited the comment.

Comment (peterskim12) [inline docs/reference/migration/migrate_5_0.asciidoc]: Ok, that makes more sense now. :) I personally don't think it's necessary -- I would hope a reader would click on the link two sentences later pointing them to more in-depth info on how they will need to re-index. This PR is more for the sake of accuracy.

Comment (nik9000) [inline docs/reference/migration/migrate_5_0.asciidoc]: Ah! Yes, that should be fine then!

**Review by nik9000 — APPROVED** [review]

**Pull request merged**